[UI] Display Connection ID in the Info Tab on Connections Page

Divyansh200102 opened the pull request:
**Notes for Reviewers**

- This PR fixes #12715

-After
![image](https://github.com/user-attachments/assets/1b957ef4-549b-441d-a46e-5a8739019076)


**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

<!--
Thank you for contributing to Meshery! 

Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits.
4. Include before and after screenshots/terminal output.

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

sudhanshutech commented on ui/components/connections/metadata.js:
```diff
@@ -199,6 +199,13 @@ const KubernetesMetadataFormatter = ({ meshsyncControllerState, connection, meta
+                  <ListItemText
+                    primary=" Connection Id"
+                    secondary={<FormatId id={connection.id} />}
+                    style={{ marginTop: '-12px' }}
```
why this styling?

Divyansh200102 replied:
For some reason the alignment of this item w.r.t other items was not right when I was making the changes I'll have a look and test the change again.

sudhanshutech replied:
@Divyansh200102 did you look? use align center or any proper css because this will create issues since if anything around it changes again the style will be affected due to negative px

Divyansh200102 replied:
@sudhanshutech i tried removing the styling entirely and this is how it's coming out.
![image](https://github.com/user-attachments/assets/58ba53b8-76b7-47ff-b4ef-6dcc09073c62)

I don't know if this is alignment issue is happening on my end but the "Updated at" and "Connection Id " are coming out at this if possible can you check this change on your end?

vishalvivekm commented:
@Divyansh200102 
Let's discuss this during the Meshery Development call today at 9:00 AM CT | 8:30 PM IST. Adding it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1#) || [meeting link](https://meet.layer5.io/meshery).

yash37158 commented:
Hi @Divyansh200102 did you make any progress ?

Divyansh200102 commented:
> Hi @Divyansh200102 did you make any progress ?

@yash37158 there are some alignment issues that seems to be coming on my end and i don't know if they are happening for others as well . Have a look at the screenshots here.In the first one i have made some alignment changes and you can see the changes in the screenshots of how it's looking after that. In the second one i removed the styling since there weren't any styling applied for other list items in the code so there shouldn't be any need of styling as the spacing and alignment bw other list items  were handled  by the grid so there shouldn't be any need to add any particular styling for this particular list item but the alignment wasn't coming right on my end so that's the reason i added it in the first place so i am not sure what to do now . you can view that as well how it's coming out.
https://github.com/meshery/meshery/pull/12800#issue-2706686384
https://github.com/meshery/meshery/pull/12800#discussion_r1868716297
